test/test_utils.rb

```diff
@@ -1,6 +1,21 @@
 require 'helper'
 
 class TestUtils < JekyllUnitTest
+  context "The \`Utils.deep_merge_hashes\` method" do
```

Comment:
I do not understand. I was following the convention used elsewhere in this file.

* [`context "The Utils.parse_date method" do`](https://github.com/jekyll/jekyll/blob/master/test/test_utils.rb#L67)
* [`context "The Utils.slugify method" do`](https://github.com/jekyll/jekyll/blob/master/test/test_utils.rb#L100)
* [`context "The Utils.add_permalink_suffix method" do`](https://github.com/jekyll/jekyll/blob/master/test/test_utils.rb#L184)
* [`context "The Utils.safe_glob method" do`](https://github.com/jekyll/jekyll/blob/master/test/test_utils.rb#L199)
* etc.

Can you explain more precisely what you are looking for?